action.yml

```diff
@@ -1,6 +1,6 @@
 name: Ultimate Portfolio Maker
```

Comment:
I would suggest do not make any change here because it may break the action.

Reply:
Yes 😅😅